Esteban-Salas-Posada/tarea-11/routes/src/pages/04/task4.js

```diff
@@ -0,0 +1,8 @@
+export default function Task4() {
+  return (
+    <>
+      <h2>Task 4</h2>
```

Comment:
@esteban-salas , this is a bad approach , you should have only one "Task" component and pass props to it